lib/adal.js

```diff
@@ -1132,15 +1249,26 @@ var AuthenticationContext = (function () {
     };
     /* jshint ignore:end */
 
+    /**
+     * Calculates the expires in value in milliseconds for the acquired token
+     * @ignore
+     */
     AuthenticationContext.prototype._expiresIn = function (expires) {
         return this._now() + parseInt(expires, 10);
     };
 
+    /**
+     * Returns the number of milliseconds
```

Comment:
Returns the current time as number of milliseconds since ___ (not sure what this value is)